fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+  $('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.staging.codacy.org/gh/stagingcodacytestuser/CheckPrCommentsPublicProjectStaging/pullRequest?prid=15436)
```suggestion
  $(".jsSelectedMainEmail").text(postVariable.email);
```